modules/apps/jax-rs/mdm-admin/src/main/java/org/wso2/carbon/mdm/api/User.java

```diff
@@ -194,6 +195,9 @@ public Response updateUser(UserWrapper userWrapper, @QueryParam ("username") Str
                 existingRolesSet.removeAll(newRolesSet);
                 // Now we have the roles to delete
                 String[] rolesToDelete = existingRolesSet.toArray(new String[existingRolesSet.size()]);
+                List<String> listRole = new ArrayList<String>(Arrays.asList(rolesToDelete));
+                listRole.remove(ROLE_EVERYONE);
```

Comment:
Variable name looks like an action. Please change this to something like "roles" or "roleList".